### DISCUSSION TITLE
需要替换nginx的相应的案例 官方文档成型的案例很少

### DISCUSSION THREAD

**Discussion by coeove** [bfenetworks/bfe#850] [Q&A (问题和回答)] [answered]
upstream api_m_pools {
    server 127.0.0.1:9975 weight=5;
}
server {
    listen       9979;
    listen       [::]:9979;
    server_name  m.app.com;
    root         /home/tomss/server/pcs/h5;
    index	 index.html;
    location 	 /loc/ {
		 proxy_set_header Host $host;
		 proxy_pass http://api_m_pools/;
    }
}

例如nginx这样的动静分离配置如何使用bfe实现？
文档缺失设计的大致概念模型 例如模型之间的关系产品线、集群、子集群、实例，他们之间的调用关系，数据流转关系。
官方文档写的很分散 案例不集中

还有就是一台服务器多个端口如何配置？

**Comment by iyangsj** [comment, marked as answer]
感谢你反馈的问题。目前的文档并不是面向熟悉nginx用户而编写，后续的版本我们将考虑更好的优化。

相关问题的答复具体如下：

>> 文档缺失设计的大致概念模型 例如模型之间的关系产品线、集群、子集群、实例，他们之间的调用关系，数据流转关系。

概念术语：https://www.bfe-networks.net/zh_cn/introduction/terminology/
转发流程：https://www.bfe-networks.net/zh_cn/introduction/forward_model/




>> 例如nginx这样的动静分离配置如何使用bfe实现

流量转发（动态流量代理）：示例参见 https://www.bfe-networks.net/zh_cn/example/route/

静态文件（静态资源响应）：这个功能需要启用mod_static模块，模块配置可参考https://www.bfe-networks.net/zh_cn/modules/mod_static/mod_static/

另外，目前的用户手册偏字典式风格供理解的用户查阅。更容易入门的材料可参考BFE书籍（https://github.com/baidu/bfe-book#%E6%93%8D%E4%BD%9C%E7%AF%87  ）




>> 还有就是一台服务器多个端口如何配置？

后端服务器配置见 https://www.bfe-networks.net/zh_cn/configuration/cluster_conf/cluster_table.data
其中通过Addr、Port字段来标识一台后端服务器实例。可以配置多个具有相同IP不同端口的后端服务器实例

**Reply by coeove** [reply]
多谢！值得研究学习一下。
